### PR TITLE
xiiui ← Body overflow hidden

### PR DESCRIPTION
**Pull request by HZooly**
## Scope

What's changed:

- Set `overflow: hidden` on `body` in `_base.scss` to prevent tooltips near screen edges from triggering unwanted scrollbars

## Potential Risks / Drawbacks

- Tooltip content that overflows the viewport edge will be clipped rather than scrollable — this will be fixed with the [new tooltip component](https://github.com/directus/directus/pull/27029)
- Since  `body` is locked to (100% of the viewport) and all scrolling handled by internal containers, this should have no impact on scrollable areas

## Tested Scenarios

- Verified that tooltips positioned near the right edge of the screen no longer cause a horizontal scrollbar to appear

## Review Notes / Questions

- I would like reviewers to pay attention to any edge case where body-level scroll might be expected (e.g. very small viewports, embedded views)

## Checklist

- [ ] Added or updated tests
- [x] Documentation PR created [here](https://github.com/directus/docs) or not required
- [x] OpenAPI package PR created [here](https://github.com/directus/openapi) or not required

---

Fixes #[CMS-2111](https://linear.app/directus/issue/CMS-2111/tooltip-causes-unwanted-scrollbars-to-appear)

### CODE REVIEW

**Review by alvarosabu — APPROVED** [review]
LGTM 🐧

File: app/src/styles/_base.scss [hunk 1]
```diff
@@ -31,6 +31,7 @@ html {
 body {
 	@include mixins.type-text;
 
+	overflow: hidden;
```
Comment (formfcw) [inline app/src/styles/_base.scss:36]: Let's make `body` a positioning context and extend the overflow lock to `html` too. `position: relative` anchors absolutely-positioned descendants inside `body`, and adding `overflow: hidden` on `html` prevents anything that escapes `body` from re-introducing a viewport scrollbar.

```suggestion
	position: relative;
	overflow: hidden;
```

Also add `overflow: hidden;` inside the `html` rule above.

I’ve tested this on FF, Chr and Safari. Without this I am still having this issue.

**Review by formfcw — CHANGES_REQUESTED** [review]
Thank you @HZooly :heart: Great work!

Really appreciate the thorough PR description for this minimal change 😄

File: app/src/styles/_base.scss (hunk 1 above)
Comment (formfcw) [reply app/src/styles/_base.scss:36]: Resolved in [2c9a013](https://github.com/directus/directus/pull/27120/commits/2c9a0139c63c3400c853a1b64a6ec3328eec5b89). @alvarosabu Please take another look, given my dual role here 🙃
